compiler/fir/checkers/gen/org/jetbrains/kotlin/fir/analysis/diagnostics/FirErrors.kt

```diff
@@ -867,6 +867,7 @@ object FirErrors : KtDiagnosticsContainer() {
     val NOT_NULL_ASSERTION_ON_CALLABLE_REFERENCE: KtDiagnosticFactory0 = KtDiagnosticFactory0("NOT_NULL_ASSERTION_ON_CALLABLE_REFERENCE", WARNING, SourceElementPositioningStrategies.OPERATOR, KtExpression::class, getRendererFactory())
     val USELESS_ELVIS: KtDiagnosticFactory1<ConeKotlinType> = KtDiagnosticFactory1("USELESS_ELVIS", WARNING, SourceElementPositioningStrategies.USELESS_ELVIS, KtBinaryExpression::class, getRendererFactory())
     val USELESS_ELVIS_RIGHT_IS_NULL: KtDiagnosticFactory0 = KtDiagnosticFactory0("USELESS_ELVIS_RIGHT_IS_NULL", WARNING, SourceElementPositioningStrategies.USELESS_ELVIS, KtBinaryExpression::class, getRendererFactory())
+    val USELESS_ELVIS_LEFT_IS_NULL: KtDiagnosticFactory0 = KtDiagnosticFactory0("USELESS_ELVIS_LEFT_IS_NULL", WARNING, SourceElementPositioningStrategies.USELESS_ELVIS, KtBinaryExpression::class, getRendererFactory())
```

Comment:
This file is generated. Please modify `compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt` and run the `Generate_FIR_Checker_Components_and_FIR_IDE_Diagnostics` configuration to generate all files.